core/src/model/Entity.java

```diff
@@ -23,6 +23,15 @@ public void setCollisionResponse(CollisionFilter collisionResponse) {
         this.collisionResponse = collisionResponse;
     }
 
+    /**
+     * Abstract constructor
```

Comment:
Abstract constructor? I think it should be written which object is created instead.

Reply:
should be "Abstract constructor for Entity" atleast